recipe/meta.yaml

```diff
@@ -37,7 +37,7 @@ requirements:
     - regex !=2019.12.17
```

Comment:
```yaml
run:
  - python
  - datasets !=2.5.0 (We should try to remove it and see what breaks)
  - filelock
  - huggingface_hub >=0.19.3,<1
  - safetensors >=0.4.1
  - tokenizers >=0.14,<0.19
  - tqdm >=4.27
```

Reply:
@psteyer Pinnings should be updated for:
```
  - huggingface_hub >=0.19.3,<1
  - safetensors >=0.4.1
```